kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -245,26 +314,18 @@ define([
                     title: span(['File Paths']),
                     name: `${cssClassType}s-area`,
                     body: [
-                        table(
-                            {
-                                class: `${cssBaseClass}__table`,
-                                dataElement: `${cssClassType}-fields`,
-                            },
-                            [
-                                tr({
-                                    class: `${cssBaseClass}__table_row`,
-                                    dataElement: `${cssClassType}-fields-row`,
-                                }),
-                            ]
-                        ),
+                        table({
+                            class: `${cssBaseClass}__table`,
+                            dataElement: `${cssClassType}-fields`,
+                        }),
                         button(
                             {
                                 class: `${cssBaseClass}__button--add_row btn btn__text`,
                                 type: 'button',
                                 id: events.addEvent({
                                     type: 'click',
-                                    handler: function (e) {
-                                        addRow(e);
+                                    handler: function () {
+                                        addRow();
```

Comment:
`addRow` used to need the event to know where the table was. We already have a handle on the table from its `data-element` attribute, so this isn't necessary. Now, it just adds a new row with default param values.